README.md

```diff
@@ -4,68 +4,95 @@ languages:
+export AZURE_CLIENT_ID="<YourAzureClientID>"
+export AZURE_CLIENT_SECRET="<YourAzureClientSecret>"
+export AZURE_TENANT_ID="<YourAzureTenantID>"
+```
 
-- open the solution in Visual Studio - NuGet should resolve the necessary packages
+Windows
 
+```cmd
+setx AZURE_CLIENT_ID "<YourAzureClientID>"
+setx AZURE_CLIENT_SECRET "<YourAzureClientSecret>"
+setx AZURE_TENANT_ID "<YourAzureTenantID>"
+```
 
-### Quickstart
-Follow these steps to get started with this sample:
+## Run the application
+First, clone the repository on your machine:
 
-1. git clone https://github.com/Azure-Samples/key-vault-dotnet-recovery.git
-2. cd key-vault-dotnet-recovery
-4. edit the app.config file, specifying the tenant, subscription, AD app id, object id and client secret
-5. dotnet run AzureKeyVaultRecoverySamples.csproj
+```bash
+git clone https://github.com/Azure-Samples/key-vault-dotnet-recovery.git
+```
 
+Then, switch to the project folder to edit the app.config file, specifying the required parameters.
+```bash
+cd key-vault-dotnet-recovery
+```
+Finally, run the application with the `dotnet run` command.
 
-## Demo
+```console
+dotnet run
+```
 
+## This sample shows how to do following operations of Azure Key Vault
+- Back up and restore Key Vault entities
+- Enable soft delete
+- Delete, recover and purge a vault
+- Delete, recover and purge vault entities
 
-## Resources
+The following samples are also related:
 
-Please see the following links for additional information:
+- [Recovery scenario samples for Azure Key Vault using the Azure Node SDK]
 
-- [Azure Key Vault soft-delete overview](https://docs.microsoft.com/azure/key-vault/key-vault-ovw-soft-delete)
-- [How to use Key Vault soft-delete with PowerShell](https://docs.microsoft.com/azure/key-vault/key-vault-soft-delete-powershell)
-- [How to use Key Vault soft-delete with CLI](https://docs.microsoft.com/azure/key-vault/key-vault-soft-delete-cli)
+## More information
 
-The following samples are also related:
+The [Azure Key Vault documentation] includes a rich set of tutorials and conceptual articles, which serve as a good complement to the samples.
+
+This project has adopted the [Microsoft Open Source Code of Conduct].
+For more information see the [Code of Conduct FAQ] or contact [opencode@microsoft.com] with any additional questions or comments.
 
-- [Recovery scenario samples for Azure Key Vault using the Azure Python SDK](https://azure.microsoft.com/resources/samples/key-vault-recovery-python/)
+<!-- LINKS -->
+[Linux]: https://dotnet.microsoft.com/download
+[Windows]: https://dotnet.microsoft.com/download
+[free account]: https://azure.microsoft.com/free/?WT.mc_id=A261C142F
+[Azure Portal]: https://portal.azure.com
+[Recovery scenario samples for Azure Key Vault using the Azure Node SDK]: https://docs.microsoft.com/en-us/samples/azure-samples/key-vault-node-recovery/key-vault-node-recovery/
+[Azure Key Vault documentation]: https://docs.microsoft.com/en-us/azure/key-vault/general/basic-concepts
```

Comment:
URLs should not include the locale. In VSCode, install the Microsoft Docs Authoring Pack for rules you should follow when writing markdowns (it would catch this issue).

I see a couple URLs like that. This always goes to the specified language/locale, while leaving it out uses the browsers' preferred languages (better for non-en-US readers).